### PR TITLE
Created password instance for each User

### PR DESCRIPTION
**Pull request by svelasco7**
Have a field to store Password for a user. This is part 1 of Proj 1.

### CODE REVIEW

File: src/main/java/codeu/controller/LoginServlet.java [hunk 1]
```diff
@@ -30,6 +30,8 @@
   /** Store class that gives access to Users. */
   private UserStore userStore;
 
+
```
Comment (dcollier) [inline src/main/java/codeu/controller/LoginServlet.java]: couple extra newlines here

File: src/main/java/codeu/model/data/User.java [hunk 2]
```diff
@@ -21,20 +21,29 @@
...
    * @param creation the creation time of this User
    */
-  public User(UUID id, String name, Instant creation) {
+  public User(UUID id, String name, String password, Instant creation) {
     this.id = id;
     this.name = name;
+    this.password = password;
     this.creation = creation;
   }
+  /**
```
Comment (dcollier) [inline src/main/java/codeu/model/data/User.java]: add a newline right above this comment to separate this from the constructor above

File: src/test/java/codeu/controller/ChatServletTest.java [hunk 3]
```diff
@@ -135,7 +135,7 @@ public void testDoPost_ConversationNotFound() throws IOException, ServletExcepti
     Mockito.when(mockRequest.getRequestURI()).thenReturn("/chat/test_conversation");
     Mockito.when(mockSession.getAttribute("user")).thenReturn("test_username");
 
-    User fakeUser = new User(UUID.randomUUID(), "test_username", Instant.now());
+    User fakeUser = new User(UUID.randomUUID(), "test_username", "password",Instant.now());
```
Comment (dcollier) [inline src/test/java/codeu/controller/ChatServletTest.java]: add a space after "password", (here and other places)

File: src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java [hunk 4]
```diff
@@ -40,13 +40,15 @@ public void tearDown() {
   public void testSaveAndLoadUsers() throws PersistentDataStoreException {
     UUID idOne = UUID.randomUUID();
     String nameOne = "test_username_one";
+    String passwordOne = "password_one";
     Instant creationOne = Instant.ofEpochMilli(1000);
-    User inputUserOne = new User(idOne, nameOne, creationOne);
+    User inputUserOne = new User(idOne, nameOne, passwordOne, creationOne);//made change
```
Comment (dcollier) [inline src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java]: you don't need to add the "//made change" comment - in the git commit history, we'll be able to look back and see that this change was associated with this commit :)

(here and other places)

**Review by dcollier — CHANGES_REQUESTED** [review]
Looks good overall - just a few style changes needed.

File: src/main/java/codeu/controller/LoginServlet.java (hunk 1 above)
Comment (svelasco7) [reply src/main/java/codeu/controller/LoginServlet.java]: Done!

File: src/main/java/codeu/model/data/User.java (hunk 2 above)
Comment (svelasco7) [reply src/main/java/codeu/model/data/User.java]: Done

File: src/test/java/codeu/controller/ChatServletTest.java (hunk 3 above)
Comment (svelasco7) [reply src/test/java/codeu/controller/ChatServletTest.java]: Done

File: src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java (hunk 4 above)
Comment (svelasco7) [reply src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java]: Done

File: src/main/java/codeu/controller/LoginServlet.java (hunk 1 above)
Comment (dcollier) [reply src/main/java/codeu/controller/LoginServlet.java]: it looks like it's still here :)

File: src/main/java/codeu/controller/LoginServlet.java [hunk 5]
```diff
@@ -76,7 +76,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     if (!userStore.isUserRegistered(username)) {
-      //TODO: add Password
```
Comment (dcollier) [inline src/main/java/codeu/controller/LoginServlet.java]: sorry, didn't mean for the TODO comments to be removed as well - these ones can stay.

**Review by dcollier — APPROVED** [review]
Couple small things, but looks good to merge overall.

File: src/main/java/codeu/controller/LoginServlet.java (hunk 1 above)
Comment (svelasco7) [reply src/main/java/codeu/controller/LoginServlet.java]: Sorry thought you meant to add extra newlines but now removed them

File: src/main/java/codeu/controller/LoginServlet.java (hunk 5 above)
Comment (svelasco7) [reply src/main/java/codeu/controller/LoginServlet.java]: Done

**Pull request merged**